local-compose.yml

```diff
@@ -0,0 +1,29 @@
+version: '2'
+services:
+    mongodb:
+        image: autopilotpattern/mongodb:latest
+        restart: always
+        mem_limit: 512m
+        build: .
+        env_file: _env
```

Comment:
We don't have this in the main `docker-compose.yml` and there's no script to generate it. What's supposed to be in here?